hugo/content/blog/2022-01-16-mumble-1-4-230.md

```diff
@@ -0,0 +1,392 @@
+date: 2022-01-16
+categories:
+  - Release
+---
+
+We are proud to present you with the first stable release of the Mumble 1.4.x release series, which brings many new features, bug fixes and general
+improvements.
+
+You can download the new version from our [**Downloads page**]({{< relref "/downloads" >}}) or the
+[**GitHub release page**](https://github.com/mumble-voip/mumble/releases/tag/v1.4.230) or from within your Windows client or software package
+management system.
+
+<!--more-->
+
+# Mumble 1.4.230
+
+It has been over two years since we released Mumble 1.3.0. Since then many new features and improvements have found their way into the software. And
+not less importantly: many fixes for issues that have been encountered with Mumble. Talking about each and every change here would definitely exceed
+the size of any reasonable release announcement. Therefore, we will restrict ourselves to highlighting what we think are the most prominent and/or
+interesting changes and we invite everyone to simply try out the new version and explore what has changed yourself. For the interested reader, we of
+course also include a full changelog at the bottom of this page as well.
+
+If you are wondering, where versions 1.4.0 to 1.4.229 went, please have a look at a recent
+[blog post]({{< relref "/blog/2021-12-07-new-versioning-scheme/index.md" >}}) of us. In their we go into the details of our _new versioning scheme_ that
+explains the absence of the mentioned version numbers.
+
+
+## Feature spotlight
+
+As there are many, many changes with this new version, here is a selection of things that we think are most prominent. If you're interested in all
+changes, have a look at the full [changelog](#changelog).
+
+### Plugin framework
+
+Mumble has supported something it called "plugins" for a very long time. However, these plugins were not what one would typically expect when hearing
+about them for the first time. Instead of being general-purpose, (potentially) multi-function plugins, they were only ever dedicated for providing
+positional data from games (generally by reading the game's in-memory representation). In addition to that, these plugins couldn't really be installed
+and updated independently from the main application (for technical reasons) and therefore, plugins were mostly a static part of the Mumble client.
+
+With Mumble 1.4 a new, general purpose plugin framework has been introduced. With this framework, the range of possibilities offered to plugin
+developers has been vastly increased and most importantly plugins are no longer restricted to positional data delivery (though that is of course still
+possible) and they can now be installed and updated at any time, without causing incompatibilities with the main Mumble client.
+
+If you want to find out more about this new plugin framework (including how to write your first plugin), check out [the
+documentation](https://github.com/mumble-voip/mumble/tree/master/docs/dev/plugins).
```

Comment:
I would label the link `the plugin documentation` for a more specific and independently specific hyperlink - which should also improve SEO.
```suggestion
plugin documentation](https://github.com/mumble-voip/mumble/tree/master/docs/dev/plugins).
```